system-test/spanner.ts

```diff
@@ -8894,6 +8894,24 @@ describe('Spanner', () => {
         mismatchedColumnError(done, DATABASE, googleSqlTable);
       });
 
+      it('GOOGLE_STANDARD_SQL should use getTransaction for executing sql', async () => {
+        const promise = await DATABASE.getTransaction({optimisticLock: true});
+        const transaction = promise[0];
```

Comment:
nit:
```suggestion
        const transaction = (await DATABASE.getTransaction({optimisticLock: true}))[0];
```

(The name `promise` seems to indicate that the returned type is a `Promise`, but it is an array.)

Reply:
Updated